.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Self update testing
+on: push
+jobs:
+  # Checkout in separate job because docker image is alpine based and checkout action doesn't work.
+  functional:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: self-update-fixture
+    container:
+      image: quay.io/pantheon-public/php-ci:v7.4
+    name: Run functional tests
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: self-update
+      - name: Checkout self-update-fixtures
+        uses: actions/checkout@v2
+        with:
+          repository: kporras07/self-update-fixture
```

Comment:
Note this is using this repo for the test fixtures. We may want to move this repo to consolidation org